### PR TITLE
Add webpack to compile app, begin component separation and add sass.

### PR DESCRIPTION
**Pull request by garethweaver**
I was hoping to help you work on the front-end of this app. I've begun the process of refactoring the front-end to split it into components. I've added sass compilation via webpack dev sever.

I've not tested this with the ledger but I shouldn't have broken anything...

Still todo:
* write a build command to package app
* finish component refactor
* update and design UI

### CODE REVIEW

**Comment by garethweaver** [comment]
I cannot for the life of me get the comm_node. The import should be this surely?

**Comment by coranos** [comment]
comm_node comes from the ledgerco package.

because the ledgerco package doesn't play nice with electrum, you have to import the js file directly:

const comm_node = require('ledgerco/src/ledger-comm-node');

**Comment by coranos** [comment]
ah, also they are cheating.
They use ledger's github:
https://github.com/timart1n/ark-desktop/blob/master/package.json#L59

I use ledger's published version, which has the electrum bug:
https://github.com/coranos/ledger-wallet-raiblocks/blob/master/package.json#L42

aha, they have a new version that fixes the bug. just need to update the package.json

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+require('./screen.sass')
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './components/home.jsx'
+
+var renderApp = () => {
```
Comment (jonathan-reisdorf) [inline src/app.js]: let

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+require('./screen.sass')
```
Comment (jonathan-reisdorf) [inline src/app.js]: would be great to end the statements with ;

File: src/components/home.jsx [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+    signature = undefined,
+    signatureInfo = undefined,
+    balance = undefined,
+    balanceInfo = undefined,
+    encodeTransactionResponse = undefined,
+    signedTransaction = undefined,
+    sentTransaction = undefined,
+    sentTransactionInfo = undefined,
+    bip44Path = undefined;
+
```
Comment (jonathan-reisdorf) [inline src/components/home.jsx]: let's refactor all of this into a class

File: src/components/_home.sass [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  display: block
+  width: 100%
+
+header
+  background: #333333
+  color: white
+  padding: 20px
+
+.container
+  padding: 20px
```
Comment (jonathan-reisdorf) [inline src/components/_home.sass]: I know you've just started, but just as a general proposal: would be great to use BEM or some other methodology regarding the naming

File: src/components/home.jsx (hunk 3 above)
Comment (jonathan-reisdorf) [reply src/components/home.jsx]: also see if you can use any of my stuff from PR #1 :)

File: src/components/_home.sass (hunk 4 above)
Comment (garethweaver) [reply src/components/_home.sass]: Sure, was just throwing some sass in there, this isn't even the UI I'd intend to build. Would looks to make this look just like the other Ledger wallets.

File: src/components/home.jsx (hunk 3 above)
Comment (garethweaver) [reply src/components/home.jsx]: This shouldn't be a class, it will be moved into a react component eventually.

**Pull request closed without merging**